aiida_vasp/data/potcar.py

```diff
@@ -748,6 +749,10 @@ def get_potcars_dict(cls, elements, family_name, mapping=None):
                 query.append(Group, filters={'type_string': OLD_POTCAR_FAMILY_TYPE, 'label': family_name}, tag='family')
                 query.append(cls, tag='potcar', with_group='family', filters=element_filters)
                 if query.count() > 1:
+                    if auto_migrate:
+                        # Migrate to new group labels, and retry
+                        migrate_potcar_group()
+                        return cls.get_potcars_dict(elements=elements, family_name=family_name, mapping=mapping, auto_migrate=False)
                     raise NotExistent(
```

Comment:
So this would then be for legacy functionality, if setting `auto_migrate=False`? Is that the idea here?

Reply:
This is to avoid multiple recursions if anything goes wrong, e.g. if the migration did not work out.